Update pd-configuration-file.md (#5726)
pingcap/docs: pd-configuration-file.md

+ The number of Region scheduling tasks performed at the same time
+ Default value: `2048`

> ### `hot-region-schedule-limit`

Reviewer: ```suggestion
### `hot-region-schedule-limit`

```

> + The number of Hot Region scheduling tasks performed at the same time. It is independent with Region scheduling.

Reviewer: ```suggestion
+ Controls the hot Region scheduling tasks that are running at the same time. It is independent of the Region scheduling.
```

+ Default value: `4`

### `hot-region-cache-hits-threshold`

> + The threshold used to set the number of minutes required to identify the hotspot region. Only after the region is in the hotspot state for more than this number of minutes can it participate in hot region scheduling

Reviewer: ```suggestion

+ The threshold used to set the number of minutes required to identify a hot Region. PD can participate in the hotspot scheduling only after the Region is in the hotspot state for more than this number of minutes.
```
